frontend/src/static/js/components/webstatus-overview-page.ts

```diff
@@ -130,16 +130,39 @@ export class OverviewPage extends LitElement {
       .then(allFeatures => {
         // Use CELL_DEFS to define the columns and
         // get the current (active) columns.
+        const columns: string[] = [];
         const columnKeys = parseColumnsSpec(getColumnsSpec(this.location));
-        const columns = columnKeys.map(
-          columnKey => CELL_DEFS[columnKey].nameInDialog
-        );
+
+        const pushBrowserName = (name: string) => {
+          columns.push(name);
+          columns.push(`${name} WPT Stable Score`);
+        };
+
+        columnKeys.forEach(columnKey => {
+          const name = CELL_DEFS[columnKey].nameInDialog;
+          switch (columnKey) {
+            case ColumnKey.Name:
+              columns.push(name);
+              break;
+            case ColumnKey.BaselineStatus:
+              columns.push(name);
+              break;
+            case ColumnKey.StableChrome:
+            case ColumnKey.StableEdge:
+            case ColumnKey.StableFirefox:
+            case ColumnKey.StableSafari:
+              pushBrowserName(name);
+              break;
+          }
+          columns.push(name);
```

Comment:
This line is causing the columns to be pushed twice.

Reply:
Oops. Thanks.